### ISSUE TITLE
pd-upscaler RE2 rt version brief white flickering on objects in left eye

### ISSUE THREAD

**Issue by nwWNTR** [praydog/REFramework#894]
There is brief white flickering on objects, walls and the floor when transitioning through areas in resident evil 2.
The flickering seems to be only in the left eye.

HTC VIVE

**Comment by rpinheiro0** [comment]
Idk about RE2 but on RE4R this is fixed by disabling Bloom.

**Comment by DaveACG** [comment]
> Idk about RE2 but on RE4R this is fixed by disabling Bloom.

Yes, this is likely the issue that has existed for quite a well and affects all RE games to varying degrees, [likely the same issue I opened a while ago. ](https://github.com/praydog/REFramework/issues/673#issue-1628161482)The mod doesn't have the ability (currently) to separate out the post-process effects and they don't play nicely with the jitter/upscaling atm. Only solution really is to disable bloom/dof/motion blur.
